test-suite/misc/non-marshalable-state/_CoqProject

```diff
@@ -1,6 +1,7 @@
 META.coq-test-suite
 -Q theories Marshal
 -I src
+-I .
```

Comment:
Why the META file is still here, and not src/, and why it all works?

Reply:
it works because this isn't testing coq_makefile